docker/validator/Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM toolchain AS builder
 COPY . /libra
 
 ENV RUSTFLAGS "-Ctarget-cpu=skylake -Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3"
-RUN cargo build --release -p libra-node -p cli -p config-builder -p libra-key-manager -p safety-rules && cd target/release && rm -r build deps incremental
+RUN cargo build --release -p libra-node -p cli -p config-builder -p libra-key-manager -p safety-rules -p db-bootstrapper && cd target/release && rm -r build deps incremental
```

Comment:
You need to copy this exact build command to the other docker files, otherwise the build will get repeated for the different images.

Reply:
Hmm. but why didn't we issue separate build commands so that different roles doesn't refer to components they don't need?

Reply:
Building everything once is faster than doing separate builds for each image.